Formula/wine.rb

```diff
@@ -430,7 +427,7 @@ def install
     if build.with? "win64"
       args64 = ["--prefix=#{prefix}"] + depflags
       args64 << "--enable-win64"
-      args64 << "--without-x" if build.without? "x11"
+      args64 << "--without-x"
 
       mkdir "wine-64-build" do
         system "../configure", *args64
```

Comment:
We could inline all args here, and avoid the `args64` variable.

Reply:
Should the option for 64-bit support also be made mandatory?

Reply:
Yes, let's make 64-bit support mandatory, with unconditional `depends_on :xcode => ["8.0", :build]`